### PR TITLE
add Pylint to Travis, Pylint complaint.

### PR DESCRIPTION
**Pull request by jinglongwu**
Feel free to add any rule that we want to ignore now to `.pylintrc`

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,4 +1,7 @@
 language: python
 python:
   - "3.6"      # current default Python on Travis CI
-script: pytest
+install: 
+  - pip install pylint
```
Comment (andysim3d) [inline .travis.yml]: 这个不用装吧。。。

Comment (jinglongwu) [reply .travis.yml]: 我试试，暂时还不work - -

Comment (andysim3d) [reply .travis.yml]: 你可能得先把现有的code lint一下

Comment (jinglongwu) [reply .travis.yml]: 嗯

Comment (jinglongwu) [reply .travis.yml]: 👌，good and clean now.

**Review by andysim3d — APPROVED** [review]

**Pull request merged**